tests/firsttests.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { test, expect } from '@playwright/test';
+
+test('Test 1: Verify login with valid credentials', async ({ page }) => {
+    await page.goto('https://practicesoftwaretesting.com/auth/login');
```

Comment:
Let's move url to the https://playwright.dev/docs/test-use-options#basic-options